### PR TITLE
Attach camera (issue #497)

### PR DESCRIPTION
**Pull request by saechimdaeki**
attach camera shortcut not using custom activity #497  

https://github.com/OpenLauncherTeam/openlauncher/pull/512 following this PR

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -31,6 +31,8 @@
     <string name="minibar_summary__mobile_network">Shortcut to the mobile network settings.</string>
     <string name="minibar_title__notification_bar">Show Notification Bar</string>
     <string name="minibar_summary__notification_bar">Expand the notification view at the top of the screen.</string>
+    <string name="minibar_title__Camera">open camera shortcut</string>
+    <string name="minibar_summary__Camera">oepn the camera shortcut</string>
```
Comment (gsantner) [inline app/src/main/res/values/strings.xml]: Make both string IDs lowercase _camera, else build issues

Comment (saechimdaeki) [reply app/src/main/res/values/strings.xml]: I just corrected it now.

**Comment by gsantner** [comment]
does it work for you as expected? if so, lets get this in

**Comment by saechimdaeki** [comment]
@gsantner It works as expected. You may try it.

**Comment by gsantner** [comment]
:tada: 
Thanks!

**Comment by saechimdaeki** [comment]
thanks for review

**Pull request merged**